modules/testsuite/cxf-tests/src/test/java/org/jboss/test/ws/jaxws/cxf/interceptors/EndpointImpl.java

```diff
@@ -58,4 +62,24 @@ public void increment()
       });
       return input + " " + cxfMessage.get(StringBuilder.class) + " " + counter.get();
    }
+   @WebMethod
+   public String echoException(String input) throws WebServiceException {
+      throw new WebServiceException("Intended Exception");
+   }
+   
+   public String getException()
+   {
+
+      StringBuffer buffer = new StringBuffer();
+      if (!JBossWSFaultListener.exceptions.isEmpty())
+      {
+         JBossWSFaultListener.exceptions.forEach((k, v) -> {
```

Comment:
"forEach" implies more than 1 exceptions.  Current formatting should provide one or more spaces or a carriage return after ";" so that the msg does not end up looking like this, k:v;k:v;k:v;